tools/go-agent/config/agent.default.yaml

```diff
@@ -118,5 +118,9 @@ plugin:
       collect_request_headers: ${SW_AGENT_PLUGIN_CONFIG_GOFRAME_COLLECT_REQUEST_HEADERS:}
       # Controlling the length limitation of all header values
       header_length_threshold: ${SW_AGENT_PLUGIN_CONFIG_GOFRAME_HEADER_LENGTH_THRESHOLD:2048}
-
+    gozero:
+      # Collect the parameters of gozero request
+      collect_request_parameters: ${SW_AGENT_PLUGIN_CONFIG_GOZERO_COLLECT_REQUEST_PARAMETERS:true}
+      # Collect the parameters of the gozero logx
+      collect_logx: ${SW_AGENT_PLUGIN_CONFIG_GOZERO_COLLECT_LOGX:true}
```

Comment:
To reduce network traffic usage and storage usage, please default to turning off parameters and logs.